apis/v1alpha1/certificate_authority.go

```diff
@@ -82,7 +82,6 @@ type CertificateAuthoritySpec struct {
 	// and CrlConfiguration (https://docs.aws.amazon.com/privateca/latest/APIReference/API_CrlConfiguration.html)
 	// types.
 	RevocationConfiguration *RevocationConfiguration `json:"revocationConfiguration,omitempty"`
-	Status                  *string                  `json:"status,omitempty"`
```

Comment:
Should we move it in the `CA.Status`? https://docs.aws.amazon.com/privateca/latest/APIReference/API_CertificateAuthority.html#privateca-Type-CertificateAuthority-Status